.yardopts

```diff
@@ -1,6 +1,7 @@
 --markup=markdown
 --asset docs/images:images
 --exclude app/controllers/api/v2/[^/]*\.rb
+--exclude app/controllers/api/v2/concerns/.*
```

Comment:
Does the documentation contain the methods provided by the concerns where they are included?